[BUG] Attachments does not work

- Your Rocket.Chat app version: 3.1.0 (210)
- Your Rocket.Chat server version: 0.69.1
- Device (or Simulator) you're running with: Iphone SE

The bug I've encountered is problem with upload taken photos, photos from library or attachments.
After click on "upload" nothing happens.

Only video is uploaded, but i can't play it (also there is no preview of this movie).

Drawings upload works fine.

[34mAPI ➔ debug post rooms.upload/:rid threw an error: Error: File size is too small (min = 1) [file-too-small] at Filter.check (/app/bundle/programs/server/packages/jalik_ufs.js:577:13) at FileUploadClass.insert (/app/bundle/programs/server/packages/rocketchat_file-upload.js:650:14) at packages/meteor.js:258:21 at DDPCommon.MethodInvocation.Meteor.runAsUser (/app/bundle/programs/server/packages/rocketchat_api.js:2276:78) at packages/dispatch_run-as-user.js:211:14 at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1186:12) at Object.Meteor.runAsUser (packages/dispatch_run-as-user.js:210:33) at Object.post (/app/bundle/programs/server/packages/rocketchat_api.js:2275:12) at Object._internalRouteActionHandler [as action] (/app/bundle/programs/server/packages/rocketchat_api.js:194:37) at Route.share.Route.Route._callEndpoint (/app/bundle/programs/server/packages/nimble_restivus.js:347:32) at /app/bundle/programs/server/packages/nimble_restivus.js:236:33 at packages/simple_json-routes.js:98:9

![img_8135](https://user-images.githubusercontent.com/679864/45716828-13520200-bb66-11e8-9041-6b9aebfc7374.PNG)

--- a/Rocket.Chat.ShareExtension/Info.plist
+++ b/Rocket.Chat.ShareExtension/Info.plist
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>XPC!</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.1.0</string>
+	<string>3.1.1</string>
 	<key>CFBundleVersion</key>
-	<string>210</string>
+	<string>214</string>
 	<key>ITSEncryptionExportComplianceCode</key>
 	<string></string>
 	<key>NSExtension</key>

--- a/Rocket.Chat/Info.plist
+++ b/Rocket.Chat/Info.plist
@@ -206,7 +206,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.1.0</string>
+	<string>3.1.1</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleURLTypes</key>
@@ -222,7 +222,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>210</string>
+	<string>214</string>
 	<key>Fabric</key>
 	<dict>
 		<key>APIKey</key>

